backend/src/main/java/org/youcancook/gobong/domain/authentication/service/TemporaryTokenService.java

```diff
@@ -0,0 +1,45 @@
+package org.youcancook.gobong.domain.authentication.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.youcancook.gobong.domain.authentication.entity.TemporaryToken;
+import org.youcancook.gobong.domain.authentication.repository.TemporaryTokenRepository;
+
+import java.time.Clock;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class TemporaryTokenService {
+
+    @Value("${token.temporary-token-expiration-seconds}")
+    private Integer temporaryTokenExpirationSeconds;
+    private final TemporaryTokenRepository temporaryTokenRepository;
+    private final Clock clock;
+
+    @Transactional
+    public String saveTemporaryToken() {
+        String token = createToken();
+        LocalDateTime expiredAt = createExpiredAt();
+        TemporaryToken temporaryToken = new TemporaryToken(token, expiredAt);
+        temporaryTokenRepository.save(temporaryToken);
+        return token;
+    }
+
+    private String createToken() {
+        return UUID.randomUUID().toString();
+    }
+
+    private LocalDateTime createExpiredAt() {
+        return LocalDateTime.now(clock).plusSeconds(temporaryTokenExpirationSeconds);
```

Comment:
```suggestion
        return clockService.now().plusSeconds(temporaryTokenExpirationSeconds);
```
위와 같이 인터페이스를 통하도록 설계합니다. 이렇게 하면 직접 구현체에 의존하는 것을 막을 수 있습니다.